config/crd/bases/mapper.imagerelocation.pivotal.io_imagemaps.yaml

```diff
@@ -3,14 +3,18 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
sigs.k8s.io/controller-runtime v0.4.0 adds support for registering readiness probes, which is very helpful to prevent the api server from sending requests to the webhook when it isn't ready.

Reply:
Yes, **but** the readiness probes are for the the core manager process and not for the webhook server portion of it. I filed https://github.com/kubernetes-sigs/controller-runtime/issues/723 for it.

Reply:
maybe there is still a race here. For the riff acceptance tests adding a ready probe on the manager process was enough to avoid errors. Especially since the webhook should not respond to requests until the ImageMap informer has synchronized, it can't hurt.

Reply:
Those features are out of scope for this PR, but I've linked to this conversation in https://github.com/pivotal/kubernetes-image-mapper/issues/5.